test/integration/rpc.js

```diff
@@ -34,499 +34,513 @@ import {
   UnknownRpcClientError
 } from '../../src/utils/errors'
 
-describe('Aepp<->Wallet', function () {
-  this.timeout(6000)
-  let node
-  let connections
-  let connectionFromWalletToAepp
-  let connectionFromAeppToWallet
-
-  before(async function () {
-    node = await Node({ url, internalUrl, ignoreVersion })
-    connections = getConnections()
-    connectionFromWalletToAepp = BrowserWindowMessageConnection({
-      connectionInfo: { id: 'from_wallet_to_aepp' },
-      self: connections.waelletConnection,
-      target: connections.aeppConnection
-    })
-    connectionFromAeppToWallet = BrowserWindowMessageConnection({
-      connectionInfo: { id: 'from_aepp_to_wallet' },
-      self: connections.aeppConnection,
-      target: connections.waelletConnection
-    })
-  })
-
-  describe('New RPC Wallet-AEPP', () => {
-    const keypair = generateKeyPair()
-    let aepp
-    let wallet
-
-    before(async () => {
-      wallet = await RpcWallet({
-        compilerUrl: compilerUrl,
-        accounts: [genesisAccount],
-        nodes: [{ name: 'local', instance: node }],
-        name: 'Wallet',
-        onConnection (aepp, { accept, deny }) {
-        },
-        onSubscription (aepp, { accept, deny }) {
-        },
-        onSign (aepp, { accept, deny, params }) {
-        },
-        onAskAccounts (aepp, { accept, deny }) {
-        },
-        onMessageSign (message, { accept }) {
-        },
-        onDisconnect (message, client) {
-          this.shareWalletInfo(
-            connectionFromWalletToAepp.sendMessage.bind(connectionFromWalletToAepp)
-          )
-        }
+const runs = [
+  { bridgeNode: false },
+  { bridgeNode: true }
+]
+
+runs.forEach((bridgeNode) => {
```

Comment:
I don't like to rerun all tests because of this feature, please cover just a specific case that you implemented, not touching other tests